apps/dh/e2e-dh/src/support/commands.ts

```diff
@@ -74,3 +76,19 @@ Cypress.Commands.add('login', (email: string, password: string) => {
     }
   );
 });
+
+Cypress.Commands.add('removeCookieBanner', () => {
+  Cypress.log({
+    displayName: 'Cookie banner',
+    message: 'Decline cookies',
+  });
+  cy.location('host').then(($host) => {
+    cy.setCookie('CookieInformationConsent', '%7B%22consents_approved%22%3A%5B%5D%7D', {
```

Comment:
suggestion: let's encode the object in order to see what it consists of :)

```suggestion
    cy.setCookie('CookieInformationConsent', encodeURIComponent('{"consents_approved":[]}'), {
```